### PR TITLE
Add NIRCam set_telescope_pointing regression test

### LINKED ISSUE #4423: Ensure old nircam regressions are covered in new system
Issue [JP-1220](https://jira.stsci.edu/browse/JP-1220) was created by Steven Crawford:

Review old miri regression tests and ensure similar coverage in new system

### PR DESCRIPTION
**Pull request by dmggh**
Add a test of the ``set_telescope_pointing`` script to the new regression test system, applied to a NIRCam TSGRISM mode exposure. All old NIRCam regression tests are now covered in the new system. Fixes #4423 and JP-1220.

### CODE REVIEW

File: jwst/regtest/test_nircam_tsgrism.py [hunk 1]
```diff
@@ -69,3 +73,30 @@ def test_nircam_tsgrism_stage3_whtlt(run_pipelines):
 
     # setdiff returns a table of length zero if there is no difference
     assert len(setdiff(table, table_truth)) == 0
+
+
+@pytest.mark.bigdata
+def test_nircam_setpointing(_jail, rtdata, fitsdiff_default_kwargs):
```
Comment (stscieisenhamer) [inline jwst/regtest/test_nircam_tsgrism.py]: Why using `_jail` instead of `jail`?

Comment (jdavies-st) [reply jwst/regtest/test_nircam_tsgrism.py]: If one wants function-scoped working directory, `_jail` and `rtdata` fixtures are appropriate.  Maybe we should rename these to something more obvious?

File: jwst/regtest/test_nircam_tsgrism.py [hunk 2]
```diff
@@ -69,3 +73,30 @@ def test_nircam_tsgrism_stage3_whtlt(run_pipelines):
...
+    # Copy original version of file to test file, which will get overwritten by test
+    input_file = rtdata.get_data("nircam/tsgrism/\
+       jw00721012001x_03103_00001-seg001_nrcalong_uncal_orig.fits")
+
+    # Get SIAF PRD database file
+    siaf_prd_loc = ['common', 'prd.db']
+    siaf_path = get_bigdata(*siaf_prd_loc)
+
+    # Call the WCS routine, using the ENGDB_Service
+    add_wcs(input_file, siaf_path=siaf_path, engdb_url=engdb_tools.ENGDB_BASE_URxL)
```
Comment (jdavies-st) [inline jwst/regtest/test_nircam_tsgrism.py]: Shouldn't this be `ENGDB_BASE_URL`?  no `x`.

File: jwst/regtest/test_nircam_tsgrism.py [hunk 3]
```diff
@@ -69,3 +73,30 @@ def test_nircam_tsgrism_stage3_whtlt(run_pipelines):
...
+    rtdata.output = input_file
+
+    rtdata.get_truth("truth/test_nircam_tsgrism_stages/
```
Comment (jdavies-st) [inline jwst/regtest/test_nircam_tsgrism.py]: Syntax error here.  Either no line break, or quotes at end and start.

**Pull request merged**